src/env.h

```diff
@@ -532,6 +532,29 @@ class AsyncRequest : public MemoryRetainer {
   bool stop_ = true;
 };
 
+class KVStore {
+ public:
+  virtual v8::Local<v8::String> Get(v8::Isolate* isolate,
```

Comment:
I wonder why this class uses v8 handles in the interface instead of std::string (or other types that store bytes)? The handles ties this class with v8 isolates.

Reply:
The main reason is that v8 `String` instances can easily be decoded to/encoded from both UTF-8 and UTF-16 (which we use on Windows), and if we went with a more abstract variant, we’d probably have to perform the conversions between the two formats ourselves. (Whether we’re using `UnionBytes` or something else).